ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -726,11 +726,34 @@ FEDataManager::spread(const int f_data_idx,
     TBOX_ASSERT(cc_data || sc_data);
 
     // Make a copy of the Eulerian data.
-    const int f_copy_data_idx = var_db->registerClonedPatchDataIndex(f_var, f_data_idx);
+    int depth;
+    if (cc_data)
+    {
+        Pointer<CellDataFactory<NDIM, double> > f_cc_fac = f_cc_var->getPatchDataFactory();
+        depth = f_cc_fac->getDefaultDepth();
+    }
+    if (sc_data)
+    {
+        Pointer<SideDataFactory<NDIM, double> > f_sc_fac = f_sc_var->getPatchDataFactory();
+        depth = f_sc_fac->getDefaultDepth();
+    }
+    const f_data_idx_map_key_type f_key{ cc_data ? 0 : 1, depth };
+    int f_copy_data_idx = -1;
+    auto it = d_f_data_idx_map.find(f_key);
+    if (it == d_f_data_idx_map.end())
```

Comment:
we can skip a line if we just do `d_f_data_idx_map.count(f_key) == 0`

Reply:
Isn't `count()` more expensive than just checking if we are at the end of the map?

Reply:
I don't think so. `count` for a `std::map` is equivalent to searching since there is only one value per key.

Reply:
I am not following this suggestion. We need to try to find the key, to get the cached index; shouldn't we go ahead and use it to see if we didn't find the key?

Reply:
Good point!